README.md

```diff
@@ -89,7 +89,7 @@ Running `cargo test` should have this test pass.
 
 You can start a Ruby project with `bundle gem rutie_ruby_example` and then once
 you change into that directory run `cargo init`.  Remove the TODOs from the gemspec
-file.  Add Rutie to the `Cargo.toml` file and define the lib type.
+file and `src/main.rs`. Add Rutie to the `Cargo.toml` file and define the lib type.
```

Comment:
This won't be true if they're making an application rather than a library.  The distinction that is important here is that you need to use a `main.rs` or a `lib.rs` and if you have both then it will only work when building the code if you specify which to use `--lib` or `--bin`.

Reply:
Oops… looked at the context.  You are correct.  Just please add “if it exists” to the end of the sentance.